docs/configuring-playbook-synapse-admin.md

```diff
@@ -24,11 +22,38 @@ After configuring the playbook, run the [installation](installing.md) command ag
 ansible-playbook -i inventory/hosts setup.yml --tags=setup-all,start
 ```
 
-
 ## Usage
 
 After installation, Synapse Admin will be accessible at: `https://matrix.DOMAIN/synapse-admin/`
 
 To use Synapse Admin, you need to have [registered at least one administrator account](registering-users.md) on your server.
 
 The Homeserver URL to use on Synapse Admin's login page is: `https://matrix.DOMAIN`
+
+### Sample configuration for running behind Traefik 2.0
+
+Below is a sample configuration for using this playbook with a [Traefik](https://traefik.io/) 2.0 reverse proxy.
+
+This an extension to Traefik config sample in [own-webserver-documentation](./configuring-playbook-own-webserver.md).
+
+```yaml
+# Don't bind any HTTP or federation port to the host
+# (Traefik will proxy directly into the containers)
+matrix_synapse_admin_container_http_host_bind_port: ""
+
+matrix_synapse_admin_container_extra_arguments:
+    # May be unnecessary depending on Traefik config, but can't hurt
+    - '--label "traefik.enable=true"'
+
+    # The Synapse Admin container will only receive traffic from this subdomain and path
+    - '--label "traefik.http.routers.matrix-synapse-admin.rule=(Host(`{{ matrix_server_fqn_matrix }}`) && Path(`{matrix_synapse_admin_public_endpoint}`))"'
+
+    # (Define your entrypoint)
+    - '--label "traefik.http.routers.matrix-synapse-admin.entrypoints=web-secure"'
+
+    # (The 'default' certificate resolver must be defined in Traefik config)
+    - '--label "traefik.http.routers.matrix-synapse-admin.tls.certResolver=default"'
+
+    # The Synapse Admin container uses port 8766 internally
+    - '--label "traefik.http.services.matrix-synapse-admin.loadbalancer.server.port=8766"'
```

Comment:
I may be confusing things, but.. shouldn't this be port `80`?

The Synapse Admin container binds to port `80` internally. We expose it on port `8766` outside when `matrix_nginx_proxy_enabled: false` (in that case we set `matrix_synapse_admin_container_http_host_bind_port: '127.0.0.1:8766'` via a statement in `group_vars/matrix_servers`).

Your example configuration explicitly disables such exposure via `matrix_synapse_admin_container_http_host_bind_port: ""`, so I don't see how port `8766` will come into play at all.

Reply:
I used this [comment](https://github.com/spantaleev/matrix-docker-ansible-deploy/issues/296#issuecomment-562819174) as a template. When traefik is used `matrix_nginx_proxy_enabled: false`. Thats why synapse admin is on port 8766 internally and traefik pointed to this port.

Reply:
It says "Traefik maps to internal port of Synapse/nginx container"

In the synapse-admin case, the internal port of synapse-admin is 80, not 8766.

I don't think pointing Traefik to 8766 will work. Have you tested this setup?

Reply:
I've tested it and it worked. Thats strange bc it shouldn't have. You are corrrect. `docker ps` shows sysnapse-admin running on port 80. I guess that bc I disabled the exposure with `matrix_synapse_admin_container_http_host_bind_port: ""` I'll change the traefik port.